src/app/common/services/fs/fs.service.ts

```diff
@@ -29,7 +30,7 @@ export class FsService {
         }).join(separator);
       }).join('\n');
 
-    const blob = new Blob([csvContent], { type: 'text/csv;charset=utf-8;' });
+    const blob = new Blob([bom, csvContent], { type: 'text/csv;charset=utf-8;' });
```

Comment:
How does this affects plain CSV files? I.e. when you do not need Excel, but you open in some app. Can it parse CSV easily still?

Reply:
[BOM](https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8) doesn't affect file content, csv is openned in all applications correctly. This is just for some apps to know that content is utf-8 because default for Excel is ANSI.

Reply:
`cat` will not show you `BOM`:
```
$ cat bomfile.csv 
id;key;instalation_site;meter_type;lock_status;conditions
```
If you need to check if there is `BOM`, you can do it with `file` command:
```
$ file bomfile.csv 
bomfile.csv: UTF-8 Unicode (with BOM) text
```
or check if with `hexdump` if there is BOM byte sequence (`0xEF,0xBB,0xBF`) on beginning :
```
$ hexdump bomfile.csv 
0000000 bbef 69bf 3b64 656b 3b79 6e69 7473 6c61
```

Reply:
OK, then all is good. Thanks a lot.